src/array.vala

```diff
@@ -2,6 +2,9 @@ using GLib;
 
 public class Vast.Array : Object
 {
+    public const ssize_t NEW_AXIS = ssize_t.MAX;
+    public const size_t DEFAULT_DIMENSION = size_t.MAX;
```

Comment:
For `NEW_AXIS` if we are to add an axis, it's just a matter of:

```
arr.redim (arr.dimension + 1);
```

This way, we don't really need any constants ;)